web/src/pages/RscCachePage/RscCachePage.tsx

```diff
@@ -0,0 +1,85 @@
+import { useEffect, useState, useRef } from 'react'
+
+import { Title } from '@tremor/react'
+
+import { Metadata } from '@redwoodjs/web'
+
+const RscCachePage = () => {
+  const webSocket = useRef<WebSocket>(null)
+  const [rscCache, setRscCache] = useState<Record<string, any>>({})
+  const [lastUpdatedKey, setLastUpdatedKey] = useState('')
+
+  useEffect(() => {
+    const socket = new WebSocket('ws://127.0.0.1:18998')
```

Comment:
Need to figure out something better for this before we can merge this PR

Reply:
Actually decided this is going to have to be good enough for now. We don't even know if we're going to keep this stuff around or not at all